### PR TITLE
Update configure-exchange-server-for-hybrid-modern-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note about ECP owa and EAS

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4448

File: microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 1]
```diff
@@ -181,6 +184,8 @@ If you are an on-premises customer using Exchange server on TCP 443, bypass traf
 52.127.96.0/23
 ```
 
+The Outlook app for iOS and Android is designed as the best way to experience Microsoft 365 or Office 365 on your mobile device by using Microsoft services to help find, plan, and prioritize your daily life and work. For more information, Please refer to [Using hybrid Modern Authentication with Outlook for iOS and Android](https://docs.microsoft.com/exchange/clients/outlook-for-ios-and-android/use-hybrid-modern-auth?view=exchserver-2019)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
The Outlook app for iOS and Android is designed as the best way to experience Microsoft 365 or Office 365 on your mobile device by using Microsoft services to help find, plan, and prioritize your daily life and work. For more information, please refer to [Using hybrid Modern Authentication with Outlook for iOS and Android](https://docs.microsoft.com/exchange/clients/outlook-for-ios-and-android/use-hybrid-modern-auth?view=exchserver-2019).
```

File: microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 2]
```diff
@@ -61,6 +61,9 @@ Turning on HMA means:
 
 Since many prerequisites are common for both Skype for Business and Exchange, review [Hybrid Modern Authentication overview and prerequisites for using it with on-premises Skype for Business and Exchange servers](hybrid-modern-auth-overview.md). Do this  *before*  you begin any of the steps in this article.
 
+> [!NOTE]
+> Outlook Wen app and Exchange Control panel does not work with HMA.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
> Outlook Web App and Exchange Control Panel does not work with hybrid Modern Authentication.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**